server/src/internalClusterTest/java/org/opensearch/cluster/coordination/ZenDiscoveryIT.java

```diff
@@ -117,6 +117,7 @@ public void testHandleNodeJoin_incompatibleClusterState() throws InterruptedExce
 
         final CompletableFuture<Throwable> future = new CompletableFuture<>();
         DiscoveryNode node = state.nodes().getLocalNode();
+
```

Comment:
[minor]: unintended change.

Reply:
This file is not in diff anymore, identical to main.